docs/pages/faq/deferred-computation.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Deferred Computation in Expo and React Native
+---
+
+import SnackInline from '~/components/plugins/SnackInline'
+
+When building an application, you will often need to perform some computation after a period of time rather than in response to some event or component lifecycle event.  Because mobile applications rely heavily on interactions and your computation shares a JS thread with updates to the UI, performant apps must be careful to schedule work in small increments and at times that will not impact your users' experience.
```

Comment:
```suggestion
When building an application, you may want to delay running code by an arbitrary amount of time, rather than in response to some event. Because mobile applications rely heavily on interactions and your computation shares a JS thread with updates to the UI, performant apps must be careful to schedule work in small increments and at times that will not impact your users' experience.
```
suggested rewording, but I think once we find a better title for the page, this section will be much easier to read so feel free to not implement this change